diff --git a/setup.py b/setup.py
index 8bc4e60..f858f2f 100644
--- widgetsnbextension-4.0.2/setup.py~	2022-08-19 14:50:26.000000000 +0000
+++ widgetsnbextension-4.0.2/setup.py	2022-08-22 16:23:50.671179607 +0000
@@ -18,7 +18,7 @@
 
 data_files_spec = [(
     'share/jupyter/nbextensions/jupyter-js-widgets', 'widgetsnbextension/static', 'extension.js*'),
-    ('etc/jupyter/nbconfig/notebook.d' , '.', 'widgetsnbextension.json')
+    ('share/jupyter/nbconfig/notebook.d' , '.', 'widgetsnbextension.json')
 ]
 
 post_develop = npm_builder(
